source/particle/property/interface.cc

```diff
@@ -516,6 +550,7 @@ namespace aspect
 
             property_list.back()->parse_parameters (prm);
           }
+        plugin_names = prop_names;
```

Comment:
Why not remove `prop_names` and read into `plugin_names` instead? More code changes, but more intuitive.